### PR TITLE
[AdminBundle] Replace deprecated getRootDir call

### PR DESCRIPTION
**Pull request by acrobat**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

### CODE REVIEW

**Review by ProfessorKuma — APPROVED** [review]
Hi @, your PR passed all our requirements.

Thank you for contributing!

File: src/Kunstmaan/AdminBundle/Helper/Security/Acl/Permission/PermissionAdmin.php [hunk 1]
```diff
@@ -232,7 +232,7 @@ public function bindRequest(Request $request)
             $user = $this->tokenStorage->getToken()->getUser();
             $this->createAclChangeSet($this->resource, $changes, $user);
 
-            $cmd = 'php ' . $this->kernel->getRootDir() . '/../bin/console kuma:acl:apply';
+            $cmd = 'php ' . $this->kernel->getProjectDir() . '/../bin/console kuma:acl:apply';
```
Comment (dannyvw) [inline src/Kunstmaan/AdminBundle/Helper/Security/Acl/Permission/PermissionAdmin.php:235]: `../` should be removed?

Comment (acrobat) [reply src/Kunstmaan/AdminBundle/Helper/Security/Acl/Permission/PermissionAdmin.php]: You're right! I've fixed the code and extended the unit test to actually cover this part

**Comment by dannyvw** [comment]
The parameter kernel.root_dir is still used in some places (https://github.com/Kunstmaan/KunstmaanBundlesCMS/search?q=kernel.root_dir&unscoped_q=kernel.root_dir). This references should be replaced also.

**Comment by acrobat** [comment]
@dannyvw I pushed a fix for all other relevant usages!

**Review by dannyvw — APPROVED** [review]

**Pull request merged**